backend/README.md

```diff
@@ -3,17 +3,24 @@ Pipelines backend.
 
 ## Building & Testing
 
-All components can be built using [Bazel](https://bazel.build/). To build
-everything under backend, run: `bazel build --action_env=PATH
---define=grpc_no_ares=true //backend/...`
+All components can be built using [Bazel](https://bazel.build/). Note, 
+make sure to install version `0.23.0` as newer versions will break. To build
```

Comment:
Could this be 0.24.0? To be consistent with the bazel version used when building backend image. https://github.com/kubeflow/pipelines/blob/3de9987b5619d1b0df8107cdc0785d7e07cb39e4/backend/Dockerfile#L1

Reply:
sure sorry, thought we were using .23 from an issue thread somewhere